neqo-transport/src/connection.rs

```diff
@@ -3223,4 +3249,214 @@ mod tests {
         assert!(matches!(pkt0, Output::Datagram(_)));
         assert_eq!(pkt0.as_dgram_ref().unwrap().len(), 1232);
     }
+
+    #[test]
+    fn cc_slow_start() {
+        let mut client = default_client();
+        let mut server = default_server();
+
+        server.set_local_tparam(
+            tp_constants::INITIAL_MAX_DATA,
+            TransportParameter::Integer(65536),
+        );
+        connect(&mut client, &mut server);
+
+        let now = now();
+
+        // Try to send a lot of data
+        assert_eq!(client.stream_create(StreamType::UniDi).unwrap(), 2);
+        assert_eq!(client.stream_send(2, &[0xaa; 100_000]).unwrap(), 65535);
+
+        qinfo!("client");
+        let mut c_tx_dgrams = Vec::new();
+        loop {
+            let pkt = client.process(None, now);
+            if pkt.as_dgram_ref().is_some() {
+                c_tx_dgrams.push(pkt.dgram().unwrap())
+            } else {
+                assert!(matches!(pkt, Output::Callback(_)));
+                break;
+            }
+        }
+
+        assert_eq!(c_tx_dgrams.len(), 11);
```

Comment:
Should we have a constant somewhere in tests for INIT_CWND (=10)?

Reply:
The tricky bit here is that initial/handshake acks have already pushed cwnd up by 630 bytes, so we actually get to send ELEVEN (10 max length and one short) packets here. So I'm thinking add that const for 10 but then add 1 in this assert and add a comment explaining why.